lib/davinci_dtr_test_kit/client_groups/must_support/dtr_smart_app_questionnaire_must_support_group.rb

```diff
@@ -23,6 +23,11 @@ class DTRSmartAppQuestionnaireMustSupportGroup < Inferno::TestGroup
       `mustSupport` elements defined in the [DTR Standard Questionnaire](http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-std-questionnaire)
       profile through visual inspection (e.g., rendering, UI behavior, or guidance).
 
+      Since there are no `mustSupport` elements defined in the [DTR Adaptive Questionnaire]('http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-adapt')
+      profile that are not present in the DTR Standard Questionnaire profile, demonstrating that the client
+      system supports DTR Standard Questionnaire `mustSupport` elements allows the user to conclude that
+      that DTR Adaptive Questionnaire `mustSupport` elements are supported.
+
```

Comment:
```suggestion
            Since there are no non-required `mustSupport` elements defined in the [DTR Adaptive Questionnaire]('http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-adapt')
      profile that are not present in the DTR Standard Questionnaire profile, demonstrating that the client
      system supports DTR Standard Questionnaire `mustSupport` elements plus an adaptive questionnaire workflow 
      is sufficient to show that DTR Adaptive Questionnaire `mustSupport` elements are supported.

```